Task5/css/header/__background-image.css

```diff
@@ -0,0 +1,27 @@
+.header__background-image {
+    position: relative;
+    z-index: -1;
+    top: 0;
+
+    display: block;
+
+    width: 100%;
+}
+
+@media screen and (min-width: 1200px) {
+    .header__background-image {
+        content: url(/images/sochi.png);
```

Comment:
Please, use `srcset` attribute on `<img>` tag for creating responsive images. `content` css property is used for different purposes.